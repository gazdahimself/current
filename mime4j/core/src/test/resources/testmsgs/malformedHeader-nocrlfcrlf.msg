Subject: this is a subject
This is an invalid header
AnotherHeader: is this an header or the first part of the body?

Body text
