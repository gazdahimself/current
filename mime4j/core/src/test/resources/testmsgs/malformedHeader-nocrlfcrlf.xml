<message>
<header>
<field>
Subject: this is a subject</field>
<field>
AnotherHeader: is this an header or the first part of the body?</field>
</header>
<body>
Body text
</body>
</message>
