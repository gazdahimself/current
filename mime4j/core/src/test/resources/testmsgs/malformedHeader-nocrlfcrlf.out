Subject: this is a subject
AnotherHeader: is this an header or the first part of the body?

Body text
